lib/kubeclient/common.rb

```diff
@@ -477,5 +482,16 @@ def http_options(uri)
 
       options.merge(@socket_options)
     end
+
+    def expired_token_message(http_code, message)
```

Comment:
What is the error returned from the server in this case? I would prefer to keep mirroring the returned exception.